### PR TITLE
Update examples and README

### PR DESCRIPTION
**Pull request by smyrick**
Move everything needed to run the `/examples` to that folder. You can now just run `rhai-test` right in the folder.

Also update README to put installer at top

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,26 @@ This project is experimental and is not a fully-supported Apollo Graph project.
...
+```
+
+This will download the latest executable, store it in a folder in your user home directory (`~/.rhai-test`), and add this folder to your `$PATH`.
+
+```sh
+# Will find config file and run your tests. Note that you will need a config file for this to work.
+rhai-test
+```
+
+Note: If this script does automatically update your `$PATH`, make sure you update it to include `~/.rhai-test/bin`
```
Comment (andrewmcgivery) [inline README.md:51]: Should this say "does **not** automatically"?

File: examples/rhai-test.config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (andrewmcgivery) [inline examples/rhai-test.config.json:1]: So one problem with moving this in here is this breaks the development flow of running `cargo run` and may also break the github actions 😓

Comment (andrewmcgivery) [reply examples/rhai-test.config.json:1]: Thinking having a copy in the root for development still in addition to this copy in examples is the way to go. :)

Comment (smyrick) [reply examples/rhai-test.config.json:1]: Oh, I see. I was making it easier to run the examples after the installer is added. Let me add it back for now

**Review by andrewmcgivery — APPROVED** [review]

**Pull request merged**